### PR TITLE
Make BigtableVersion use milliseconds

### PR DESCRIPTION
**Pull request by evildour**

### CODE REVIEW

File: apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.Tests/BigtableVersionTest.cs [hunk 1]
```diff
@@ -39,28 +39,30 @@ public void Comparisons()
...
         [Fact]
-        public void MicrosScaleFromDateTime()
+        public void ScaleFromDateTime()
         {
             var utcNow = DateTime.UtcNow;
             var v1 = new BigtableVersion(utcNow);
             var v2 = new BigtableVersion(utcNow + TimeSpan.FromSeconds(1));
             var diff = v2.Value - v1.Value;
+            Assert.Equal(1_000, diff);
+            diff = v2.Micros - v1.Micros;
```
Comment (jskeet) [inline apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.Tests/BigtableVersionTest.cs]: I think this chunk of code would be clearer as

```csharp
Assert.Equal(1_000, v2.Value - v1.Value);
Assert.Equal(1_000_000, v2.Micros - v1.Micros);
```

File: apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableVersion.cs [hunk 2]
```diff
@@ -23,74 +23,93 @@ namespace Google.Cloud.Bigtable.V2
     /// <remarks>
     /// <para>
     /// Note: when using ReadModifyWriteRow, modified columns automatically use a server version, which
```
Comment (jskeet) [inline apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableVersion.cs]: I think it's probably worth explaining why both millis and micros are referred to here at all. I'm certainly confused about it, having only paid half attention to what's going on.

Is micros the raw underlying format, but the server only ever sets values in millis??

File: apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.Tests/BigtableVersionTest.cs (hunk 1 above)
Comment (evildour) [reply apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.Tests/BigtableVersionTest.cs]: Done

File: apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableVersion.cs (hunk 2 above)
Comment (evildour) [reply apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableVersion.cs]: That's correct. Ok, I'll update all these comments.

Comment (evildour) [reply apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableVersion.cs]: Actually, I think I'll make another note preceding this one to explain the whole thing.

**Comment by evildour** [comment]
Thanks Jon! PTAL

**Pull request merged**